### PR TITLE
[Changelog] add new header to main changelog page

### PR DESCRIPTION
**Pull request by ryancurtis1**
Large:
Screenshot 2024-10-18 at 2 07 39 PM

Medium:
Screenshot 2024-10-18 at 2 08 16 PM

Small:
Screenshot 2024-10-18 at 2 08 41 PM

### CODE REVIEW

File: pages/_meta.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ export default {
         "title": "Changelog",
         "type": "page",
         "theme": {
-            "sidebar": false
+            "sidebar": false,
+            layout: 'full'
```
Comment (ryancurtis1) [inline pages/_meta.ts:18]: This also effects the layout on the individual post pages but will update the layout of those pages separately

File: pages/changelogs.mdx [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+      className="changelogTitle text-4xl tracking-tighter text-center font-extrabold
 md:text-5xl mt-8 pb-6"
->
-  Changelog
-</h1>
+    >
+      Changelog
+    </h1>
+    <span>
+      <a>Subscribe to our newsletter</a> to get product updates in your inbox.
```
Comment (ryancurtis1) [inline pages/changelogs.mdx:13]: Adding the functionality to subscribe will be added separately

**Comment by junhouse** [comment]
Is this expected behavior on large screen? The new graphic is super far away.
Screenshot 2024-10-18 at 3 23 43 PM

**Comment by ryancurtis1** [comment]
> Is this expected behavior on large screen? The new graphic is super far away. 

@junhouse Yes? Let me double check with Tara, but on the Figma the graphic is always touching the right side of the screen for large screen sizes.

**Review by junhouse — APPROVED** [review]
lgtm

**Pull request merged**